bitnami/postgresql/templates/primary/statefulset.yaml

```diff
@@ -532,7 +532,7 @@ spec:
             {{- end }}
           {{- end }}
           env:
-            {{- $database := required "In order to enable metrics you need to specify a database (.Values.auth.database or .Values.global.postgresql.auth.database)" (include "postgresql.v1.database" .) }}
+            {{- $database := include "postgresql.v1.database" . }}
```

Comment:
The read statefulset has exactly [the same line](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/templates/read/statefulset.yaml#L435) and probably same changes are required. Could you give it a glance?

Reply:
How have you done it?
The main issue is that I do not want to create databases on initialization and when you do not mention any databases the helm chart "required" statement won't let you install the chart, moreover, its kinda weird you monitor any database because the URI is dependent in the .Values.auth.database value

Reply:
Hi, i have looked it further and i have committed another fix, would be awesome if you look at it.



COMMIT_MESSAGE:

Looks like there is no usage of the DB NAME part in the DATA_SOURCE_URI environment variable which means there is no need for the $database part, the postgres exporter always monitoring all the dbs, also if created after the initialization and not mentioned in the DATA_SOURCE_URI env var.

In conclusion the PR is for two reasons.
1: making DATA_SOURCE_URI env var more "right" and related to what it actually does
2: More importantly, removing the required statement in the $database helm variable assignment in order for users to be able to deploy the postgres exporter although not assigning .Values.auth.database variable for the db creation in the initialization of the postgres

Reply:
I agree, is a little bit weird. You can deploy version 15.5.10 setting `metrics.enabled=true` and `auth.database=test` and `postgresql-exporter` will monitor all the databases in the postgresql server, including the new ones after the deploy. As you said the database name in the URI seems not doing anything and I think we shouldn't require `auth.database` to set it.